app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -1,3 +1,4 @@
+# rubocop:disable Metrics/ClassLength
```

Comment:
Yeah, I didn't see any big chunks at first glance that seemed like they should be broken out.  But I figure it's worth thinking about.